autoflake.py

```diff
@@ -904,6 +906,9 @@ def fix_code(
     if not source:
         return source
 
+    if source.lstrip().startswith(IGNORE_COMMENT):
```

Comment:
can you please make sure this works with shebangs too?

Reply:
sorry, what do you mean by shebangs?

Reply:
Something like:


```python
#!/usr/bin/env python3

# autoflake: skip_file

...
```

Reply:
Done, thank you!